content/cloud/custom-fermyon-subdomain.md

```diff
@@ -0,0 +1,62 @@
+title = "Apply Custom Fermyon Subdomain"
+template = "cloud_main"
+date = "2022-04-26T00:22:56Z"
+enable_shortcodes = true
```

Comment:
We do not need shortcodes on this document. We don't appear to be using any. This line can be safely removed.

Reply:
set to 'false'